docs/storefront/debugging.md

```diff
@@ -0,0 +1,51 @@
+# Storefront Debugging
+
+This guide explains how to set up and use debugging tools for the Shopsys Platform Storefront application.
+
+## Chrome DevTools Setup
+
+To debug the Storefront application, ensure it is running in Docker. Thanks to the updated npm script:
+
+```json
+"dev": "NODE_OPTIONS='--inspect=0.0.0.0:9229' next dev"
+```
+
+the Storefront is already running locally in development mode with the Node.js inspector enabled on port 9229. No extra step is needed if you are using the default Docker setup.
+
+2. **Open Chrome DevTools:**
+   - Open Chrome and navigate to `chrome://inspect/#devices`
+   - You will see the Devices page:
+
+     ![Chrome DevTools Devices page](./images/01_node-debug_inspect-devices.png)
+
+   - Click on "Configure..." under "Target discovery settings"
+   - In the popup, add `127.0.0.1:9229` & `127.0.0.1:9230` to the list of targets:
+
+     ![Node.js Debug Configuration](./images/02_node-debug_configuration.png)
+
+   - You can also open the dedicated DevTools for Node:
+
+     ![Open dedicated DevTools for Node](./images/03_node-debug_dedicated-devtools.png)
+
+3. **Inspect your application:**
+   - You should now see your Storefront application listed under "Remote Target" in Chrome DevTools
+   - Click on "inspect" to open the DevTools window
+   - You can now set breakpoints, inspect variables, and debug your application
+
+## More Details About Why Both Ports Are Opened
+
+The Docker configuration exposes both ports 9229 and 9230 for debugging purposes. Here's the technical reason:
+
+When Node.js is started with the `--inspect=0.0.0.0:9229` option, Next.js internally spawns a second Node.js instance and automatically increments the port number by 1, resulting in port 9230 being used for the actual debugging session where you can inspect your Next.js backend application files.
+
+Both ports are opened proactively because:
+- **Port 9229**: The initial inspector port that may be used directly if Next.js changes its internal behavior in future versions
```

Comment:
This is not rendered properly - empty line should be between list and previous text
![Snímek obrazovky 2025-06-09 v 9 03 13](https://github.com/user-attachments/assets/fe229e91-b121-4384-b49f-c064f268331a)